lib/charms/data_platform_libs/v0/data_interfaces.py

```diff
@@ -3051,7 +3051,7 @@ def _on_relation_created_event(self, event: RelationCreatedEvent) -> None:
 
         # Sets topic, extra user roles, and "consumer-group-prefix" in the relation
         relation_data = {
-            f: getattr(self, f.replace("-", "_"), "")
+            f: getattr(self.relation_data, f.replace("-", "_"), "")
```

Comment:
**question:** I recall you mentioning this in a meeting, what was this for again?

Reply:
This fix: https://github.com/canonical/data-platform-libs/pull/163